metricbeat/scripts/mage/docs_collector.go

```diff
@@ -253,6 +253,10 @@ func gatherMetricsets(modulePath string, moduleName string, defaultMetricSets []
 		if !isMetricset {
 			continue
 		}
+		metricsetDoc, err := ioutil.ReadFile(filepath.Join(metricset, "_meta/docs.md"))
```

Comment:
```suggestion
		metricsetDoc, err := os.ReadFile(filepath.Join(metricset, "_meta/docs.md"))
```
since `ioutil` is deprecated

Reply:
Nice catch. Thanks